### DISCUSSION TITLE
COM port change

### DISCUSSION THREAD

**Discussion by jriker1** [zwave-js/zwave-js-ui#3374] [General]
This may be a lot but still curious if anyone knows what might be happening.

I have been using ZWaveJS UI on a Windows server and it's been fine.  Been running on COM5.  The a lot part is I did some BIOS updates to my x670e system, did some tweaks to the power settings in the bios for the 7950x CPU, and also did some Windows updates.  After that none of the Z-Wave devices were showing in HomeAssistant.  Noticed in Device Manager that it had switched to COM3.  Since I don't restart this system much is this an expectation of rebooting that the COM port would change?

**Comment by kpine** [comment]
Probably need to disable soft-reset in the Z-Wave settings.

https://github.com/zwave-js/node-zwave-js/issues/6341
